test/Microsoft.DotNet.Tools.Tests.Utilities/TestBase.cs

```diff
@@ -68,6 +69,20 @@ protected static TestAssetsManager GetTestGroupTestAssetsManager(string testGrou
             return testAssetsMgr;
         }
 
+        protected static TestSetup CreateTestSetup(
+            string testGroup,
+            string projectName,
+            [CallerMemberName] string callingMethod = nameof(CreateTestSetup),
```

Comment:
[CallerMemberName] will have precedence here - default value is required when you use CallerMemberName